app-admin/filebeat/files/filebeat.initd

```diff
@@ -12,11 +12,12 @@ FILEBEAT_OPTS="${FILEBEAT_OPTS:-}"
 command="/usr/bin/filebeat"
 command_args="-c ${FILEBEAT_CONFIG} ${FILEBEAT_OPTS} -path.config $(dirname $FILEBEAT_CONFIG) \
 	-path.data ${FILEBEAT_DATADIR} -path.home ${FILEBEAT_DATADIR} -path.logs ${FILEBEAT_LOGDIR}"
-extra_commands="checkconfig"
 command_background="true"
-start_stop_daemon_args="--user ${FILEBEAT_USER}:${FILEBEAT_GROUP} \
-	--chdir ${FILEBEAT_DATADIR}"
-pidfile="/run/filebeat/filebeat.pid"
+command_user="${FILEBEAT_USER}"
```

Comment:
You dropped `${FILEBEAT_GROUP}`, it used to be part of the `--user` flag but it's not in `command_user`.